js/ui/text_box/action_button_collection/index.js

```diff
@@ -61,11 +137,7 @@ export default class ActionButtonCollection {
     getButton(buttonName) {
         const button = find(this.buttons, ({ name }) => name === buttonName);
 
-        if(!button) {
-            throw "Cannot find button with this name";
-        }
-
-        return button.instance;
+        return button ? button.instance : null;
```

Comment:
`return button && button.instance;`

Reply:
i think "null" is better then "undefined" in this case